Automattic/wp-calypso
# Update/fix coverage reports

Resolves problems with generating test coverage reports and also submits them to coverage reporting service.

Discussion:
p4TIVU-8XJ-p2

Test plan:
1. Make sure tests are passing
1. See if there's a code coverage check under this PR
